packages/framework/src/Migrations/Version20180611215810.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20180611215810 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->sql('ALTER TABLE category_domains RENAME COLUMN hidden TO enabled');
```

Comment:
This looks like it negates all current category visibility.

Reply:
Good point I added UPDATE migration that will solve this problem.